<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>256</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightblue;
</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>First Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="F">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>12</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Second Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="S">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>12</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="F_2">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>12</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>390</width>
     <height>47</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ADD">
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SUB">
      <property name="text">
       <string>SUB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MUL">
      <property name="text">
       <string>MUL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DIV">
      <property name="text">
       <string>DIV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DIV_2">
      <property name="text">
       <string>MOD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DIV_3">
      <property name="text">
       <string>EXP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DIV_4">
      <property name="text">
       <string>FLO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
